pvnet/models/multimodal/multimodal.py

```diff
@@ -241,22 +304,45 @@ def forward(self, x):
                 modes["pv"] = self.pv_encoder(x)
             else:
                 # Target is PV, so only take the history
-                pv_history = x[BatchKey.pv][:, : self.history_len_30].float()
-                modes["pv"] = self.pv_encoder(pv_history)
+                # Copy batch
+                x_tmp = x.copy()
+                x_tmp[BatchKey.pv] = x_tmp[BatchKey.pv][:, : self.history_len + 1]
```

Comment:
Are you sure about this +1? That would give it the PV data at exactly t0 right?

Reply:
See my comment above, it should give the t0 to the input, but that I think is how the current one works for PV. I think there probably shouldn't be the +1, but didn't want to change the current behaviour.

Reply:
The live data gets updated every 3 mins, with about a 10 minute delay. 
I think this is fine, and in production we can do a small bit of interpolation. We can always change this later